frida_mode/src/entry.c

```diff
@@ -10,7 +11,7 @@
 #include "instrument.h"
 #include "persistent.h"
 #include "ranges.h"
-#include "seccomp.h"
+
```

Comment:
I'd try copying the approach for __APPLE__ for seccomp support. That way only the feature is changed depending on the OS, rather than the various points where it is integrated with the wider instrumentation engine.